lib/services/elasticsearch.js

```diff
@@ -1438,9 +1436,7 @@ class ElasticSearch extends Service {
 
     // @todo should be done in clientAdapter
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)